packages/platforms/netberg/vendor-config/src/python/netberg/__init__.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/python
+
+from onl.platform.base import *
+
+class OnlPlatformNetberg(OnlPlatformBase):
```

Comment:
Please inherit from one of the existing PortConfig classes as well. 
See https://github.com/opencomputeproject/OpenNetworkLinux/blob/master/packages/platforms/accton/x86-64/x86-64-accton-as5712-54x/platform-config/r0/src/python/x86_64_accton_as5712_54x_r0/__init__.py#L4